src/targets/Targets.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Minor nitpick, but I prefer to have `TargetFramework` be the first property.

Reply:
I prefer alphabetical order, since the list of properties can get quite big sometimes and I'd like a deterministic way to eyeball a specific property. I'm not hung up on this though. I can change it if you feel strongly enough about it.

Reply:
If you look at the other Particular repos, having `TargetFramework` first is something I've standardized on.